misc/python/materialize/cli/ci_annotate_errors.py

```diff
@@ -454,7 +454,17 @@ def annotate_logged_errors(
     step_key: str = os.getenv("BUILDKITE_STEP_KEY", "")
     buildkite_label: str = os.getenv("BUILDKITE_LABEL", "")
 
-    (known_issues, issues_with_invalid_regex) = get_known_issues_from_github()
+    known_issues = []
+    issues_with_invalid_regex = []
+    for repo in [
+        "MaterializeInc/materialize",
+        "MaterializeInc/incidents-and-escalations",
+    ]:
```

Comment:
Can we move and reuse the definition of `GROUP_REPO` from `misc/python/materialize/cli/ci_closed_issues_detect.py`?